coderedcms/templates/coderedcms/includes/iframe_gmap.html

```diff
@@ -1,7 +1,7 @@
 {% if address %}
 <div class="my-2">
     <div class="embed-responsive embed-responsive-16by9 mb-5">
-        <iframe class="embed-responsive-item" width="100%" style="border:0" src="https://maps.google.com/maps?q={{ address }}&output=embed" allowfullscreen></iframe>
+        <iframe class="embed-responsive-item" width="100%" style="border:0" src="https://maps.google.com/maps?q={{ address }}&output=embed" title="Map of {{self.address}}" allowfullscreen></iframe>
```

Comment:
`self.address` might not be reliable, since this is an include and someone could use this and pass in any address they want manually. You might be better off with `address` since that is how it is being used elsewhere.